core/src/main/scala/org/apache/pekko/persistence/jdbc/state/DurableStateQueries.scala

```diff
@@ -46,6 +46,7 @@ import pekko.persistence.jdbc.config.DurableStateTableConfiguration
   lazy val sequenceNextValUpdater = slickProfileToSchemaType(profile) match {
     case "H2"       => new H2SequenceNextValUpdater(profile, durableStateTableCfg)
     case "Postgres" => new PostgresSequenceNextValUpdater(profile, durableStateTableCfg)
+    case "MySQL"    => new MySQLSequenceNextValUpdater(profile, durableStateTableCfg)
     case _          => ???
```

Comment:
`case _ => ???`
Can we provide a default implementation，it is another feature

Reply:
unfortunately - this support is not standardised across all db servers - we will need specific versions for different db types